packages/bitcore-wallet-service/src/lib/expressapp.ts

```diff
@@ -712,6 +725,18 @@ export class ExpressApp {
     });
 */
 
+    router.post('/v4/txproposals/:id/signatures/', (req, res) => {
+      getServerWithAuth(req, res, server => {
+        req.body.txProposalId = req.params['id'];
+        req.body.maxTxpVersion = 3;
```

Comment:
no needed anymore?

Reply:
I think the new URL shiuld have a flag :"bchSchnorr' support = true.

So at the moment we are about to sign, we can check: if txp.coin=='bch' && txp.signningMethod=='schorr' && ! opts.bchSchnorr) => need to upgrade